src/error.c

```diff
@@ -157,7 +158,7 @@ yaml_error(const yaml_node_t* node, GError** error, const char* msg, ...)
     if (node != NULL) {
         error_context = get_syntax_error_context(node->start_mark.line, node->start_mark.column, error);
         g_set_error(error, G_MARKUP_ERROR, G_MARKUP_ERROR_PARSE,
-                    "%s:%zu:%zu: Error in network definition: %s\n%s",
+                    "%s:%zu:%zu: Error in network definition: %s\n%s",  // netplan-feature: error_context_lineno
```

Comment:
Should we require stricter parsing, and enforce the full comment string as '/* netplan-feature: <token> */'?  (the mix of C and C++ style comments here made me go double-check the parser)